src/components/app/app.component.js

```diff
@@ -0,0 +1,8 @@
+import React, { Component } from 'react';
+import './app.component.scss';
```

Comment:
you can add some global styles that way in `index.js`, but for components better use `css-modules`
And please configure `css-modules` in webpack(they wrote that modules are enabled in css-loader by default but seem like they're not)